src/SDKs/EventHub/Management.EventHub/generate.ps1

```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "eventhub/resource-manager" -AutoRestVersion "latest"
+Start-AutoRestCodeGeneration -ResourceProvider "eventhub/resource-manager" -AutoRestVersion "latest" -ConfigFileTag "package-2018-01-preview"
```

Comment:
nit info: Please make sure that the next time you regenerate the code you apply the correct tags here

Reply:
Sure Will make note of it.